assets/blocks/quiz/quiz-block/quiz-settings.js

```diff
@@ -71,7 +67,7 @@ const QuizSettings = ( { attributes: { options = {} }, setAttributes } ) => {
 						onChange={ createChangeHandler( 'autoGrade' ) }
 						label={ __( 'Auto Grade', 'sensei-lms' ) }
 						help={ __(
-							'Grades quiz and displays answer explanation immediately after completion. Only applicable if quiz is limited to Multiple Choice, True/False and Gap Fill questions. Questions that have a grade of zero are skipped during autograding.',
+							'Only applicable for Multiple Choice, True/False and Gap Fill questions. Questions with a grade of 0 are skipped.',
```

Comment:
I think only this part may be a little confusing: "Questions with a grade of 0 are skipped.".

Maybe complementing with the previous text would make it more clear? "... during autograding."

Reply:
How about something like `Automatically grade Multiple Choice, True/False and Gap Fill questions that have a grade above 0 set.` I think what Auto Grade does could use this little bit of extra explanation.

Reply:
What about something like https://github.com/Automattic/sensei/pull/4000/commits/e219218e8e267e9f21cc3844167d1da52f06a636?

Reply:
Looks good to me! =)